README.md

```diff
@@ -41,6 +41,43 @@ You can start editing the page by modifying `pages/index.tsx`. The page auto-upd
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+### Swim Lanes API Endpoints
+
+The following endpoints are currently available at http://localhost:3000/api/lanes/
+
+- [/books-and-periodicals](http://localhost:3000/api/lanes/books-and-periodicals)
+- [/fliers-and-ephemera](http://localhost:3000/api/lanes/fliers-and-ephemera)
+- [/manuscripts-and-correspondence](http://localhost:3000/api/lanes/manuscripts-and-correspondence)
+- [/maps](http://localhost:3000/api/lanes/maps)
+- [/photographs](http://localhost:3000/api/lanes/photographs)
+- [/prints-and-drawings](http://localhost:3000/api/lanes/prints-and-drawings)
+- [/recently-digitized-collections](http://localhost:3000/api/lanes/recently-digitized-collections)
+
+
+### Individual Lane - GET /api/lanes/[...slug]
+---
+#### Query Parameter
```

Comment:
Do you think we need to make a distinction between URL query parameters (`api/lanes?slug=books-and-periodicals`) and the way nextjs defines query parameters (`api/lanes/books-and-periodicals`)?